doc/rados/operations/index.rst

```diff
@@ -18,6 +18,7 @@ and, monitoring an operating cluster.
 	monitoring
 	monitoring-osd-pg
 	user-management
+    pg-repair
```

Comment:
ah this is <tabs> vs spaces. I thought I had changed this to be an actual tab. This is the other thing you wouldn't want to review if I had automatically switched all tabs to spaces :D